# Message type to hold a map from another robot
uint8 robot_id
nav_msgs/OccupancyGrid map
